tests/Tools/Console/Command/DumpSchemaCommandTest.php

```diff
@@ -62,6 +65,16 @@ public function testExecute(): void
             ->method('getMigrations')
             ->willReturn(new AvailableMigrationsSet([]));
 
+        $classNameGenerator = $this->createMock(ClassNameGenerator::class);
+        $classNameGenerator->expects(self::any())
```

Comment:
I don't think `->expects(self::any())` produces any effect. Please remove it here and below.